dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/tools/AbstractJspModC.java

```diff
@@ -277,6 +274,23 @@ private int _internalCompileApp(boolean doTraceCheck) throws JspCoreException {
                 if (booleanProperty != null) {
                     options.setUseJikes(booleanProperty.booleanValue());
                 }
+                String javaSourceLevel = (String) this.optionOverrides.get(JspToolsOptionKey.javaSourceLevelKey);
+                int javaLevel = -1;
+                if (javaSourceLevel != null) {
+                    if (javaSourceLevel.equals("1.8")) javaSourceLevel="8";
+                    try {
+                       javaLevel = Integer.parseInt(javaSourceLevel);       
+                    } catch (NumberFormatException e) {
+                        if (logger!=null) {
+                            if(doTraceCheck&&logger.isLoggable(Level.WARNING)){
+                                logger.logp(Level.WARNING,CLASS_NAME,"_internalCompileApp", "failed to parse javaSourceLevel: " + javaSourceLevel, e);
```

Comment:
Why are these warning messages not translated?

Reply:
This class will be removed in PR #26285, so I'll leave it alone.